keras_autodoc/get_signatures.py

```diff
@@ -21,7 +20,15 @@ def get_signature_start(function):
 
 
 def get_signature_end(function, class_aliases={}):
-    signature_end = Signature(function).format_args()
+    signature = inspect.signature(function)
+    signature = signature.replace(
+        parameters=[
+            param.replace(annotation=inspect.Signature.empty)
+            for param in signature.parameters.values()
+        ],
+        return_annotation=inspect.Signature.empty
+    )
+    signature_end = str(signature)
```

Comment:
Maybe instead we can use this: https://github.com/sphinx-doc/sphinx/blob/3.x/sphinx/util/inspect.py#L617 with `show_annotations=False`?

Reply:
Makes sense, changed in 2ab0d2f.